xml/System.IO/Path.xml

```diff
@@ -843,18 +843,21 @@ Unlike the string overload, this method doesn't normalize directory separators.
       </Parameters>
       <Docs>
         <param name="path">The path of a file or directory.</param>
-        <summary>Returns the directory information for the specified path string.</summary>
+        <summary>Returns the directory information for the specified path.</summary>
         <returns>Directory information for <paramref name="path" />, or <see langword="null" /> if <paramref name="path" /> denotes a root directory or is null. Returns <see cref="F:System.String.Empty" /> if <paramref name="path" /> does not contain directory information.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- In most cases, the string returned by this method consists of all characters in the path up to but not including the last <xref:System.IO.Path.DirectorySeparatorChar> or <xref:System.IO.Path.AltDirectorySeparatorChar>. If the path consists of a root directory, such as "c:\\", null is returned. Note that this method does not support paths using "file:". Because the returned path does not include the <xref:System.IO.Path.DirectorySeparatorChar> or <xref:System.IO.Path.AltDirectorySeparatorChar>, passing the returned path back into the <xref:System.IO.Path.GetDirectoryName%2A> method will result in the truncation of one folder level per subsequent call on the result string. For example, passing the path "C:\Directory\SubDirectory\test.txt" into the <xref:System.IO.Path.GetDirectoryName%2A> method will return "C:\Directory\SubDirectory". Passing that string, "C:\Directory\SubDirectory", into <xref:System.IO.Path.GetDirectoryName%2A> will result in "C:\Directory".  
-  
- For a list of common I/O tasks, see [Common I/O Tasks](/dotnet/standard/io/common-i-o-tasks).  
-  
-   
-  
+## Remarks
+
+In most cases, the string returned by this method consists of all characters in the path up to, but not including, the last directory separator character(s). A directory separator character can be either <xref:System.IO.Path.DirectorySeparatorChar> or <xref:System.IO.Path.AltDirectorySeparatorChar>. If the path consists of a root directory, such as "c:\\", `null` is returned.
+
+This method does not support paths using "file:".
+
+Because the returned path does not include the last directory separator character(s), passing the returned path back into the <xref:System.IO.Path.GetDirectoryName%2A> method truncates one folder level per subsequent call on the result string. For example, passing the path "C:\Directory\SubDirectory\test.txt" into <xref:System.IO.Path.GetDirectoryName%2A> returns "C:\Directory\SubDirectory". Passing that string, "C:\Directory\SubDirectory", into <xref:System.IO.Path.GetDirectoryName%2A> returns "C:\Directory".
```

Comment:
```suggestion
Because the returned path does not include the last directory separator character(s), passing the returned path back into the <xref:System.IO.Path.GetDirectoryName%2A> method truncates one folder level per subsequent call on the result path. For example, passing the path "C:\Directory\SubDirectory\test.txt" into <xref:System.IO.Path.GetDirectoryName%2A> returns "C:\Directory\SubDirectory". Passing that path, "C:\Directory\SubDirectory", into <xref:System.IO.Path.GetDirectoryName%2A> returns "C:\Directory".
```